src/data/data.ts

```diff
@@ -111,22 +115,19 @@ export const Distance: { [key: string]: FormValue } = {
   sixFt: { label: '6+ feet apart', multiplier: 0.5 },
   tenFt: { label: '10+ feet apart', multiplier: 0.25 },
 }
+
+const noneLabel = 'No mask or poorly-worn mask'
+const basicLabel = 'Cotton mask, bandana, or buff'
+const filteredLabel = 'Surgical mask, mask with filter insert, or N95'
```

Comment:
I would remove "or N95" -- we strongly believe N95 is much better than 2x wearer protection, putting it in as 2x seems confusing to me

Reply:
Done